src/pages/game/active-game.jsx

```diff
@@ -25,7 +27,7 @@ export function ActiveGame({ playerId, gameId }) {
           setError(true);
         } else {
           setError(false);
-          setGame(data.game);
+          setGame(Object.assign(data.game, { votingType }));
```

Comment:
we'll have to assign this through the socket, can't update the local state without broadcasting it to the other players